meinberlin/apps/votes/admin.py

```diff
@@ -6,14 +6,28 @@
 
 @admin.register(VotingToken)
 class VotingTokenAdmin(admin.ModelAdmin):
-    fields = ("token", "module", "is_active", "allowed_votes")
-    readonly_fields = ("token",)
+    fields = (
+        "token",
+        "token_hash",
+        "module",
+        "is_active",
+        "package_number",
```

Comment:
maybe also add package_number to read_only fields? if someone changes that manually, that breaks stuff, no?